indico/modules/events/editing/operations.py

```diff
@@ -283,3 +289,47 @@ def update_review_condition(condition, file_types):
 def delete_review_condition(condition):
     logger.info('Review condition %r deleted by %r', condition, session.user)
     db.session.delete(condition)
+
+
+def assign_editor(editable, editor):
+    editable.editor = editor
+    db.session.flush()
+    logger.info('Editable %r assigned to %r editor', editable, editor)
+
+
+def unassign_editor(editable):
+    editor = editable.editor
+    editable.editor = None
+    db.session.flush()
+    logger.info('Editor %r has been unassigned from %r', editor, editable)
+
+
+def generate_editables_zip(editables):
+    buf = BytesIO()
+    with ZipFile(buf, 'w', allowZip64=True) as zip_file:
+        for editable in editables:
+            for revision_file in editable.revisions[-1].files:
+                file_obj = revision_file.file
+                with file_obj.storage.get_local_path(file_obj.storage_file_id) as src_file:
```

Comment:
`with file_obj.get_local_path() as src_file` (let's fix this in `RHExportRevisionFiles` and `ZipGeneratorMixin` too)